placideirandora/epic-mail
# #164852002 implemented the last lfas' feedback and mine - code refactoring

#### What does this PR do?
It implements the last LFA's feedback and mine.

#### Description of Task to be done?
1. It confuses hackers by replying with 'invalid email or password'.
2. It allows users to create groups with the same name but uniquely in their groups.
3. It comments the code.
4. It updates the readme.
5. It refactors the code.

#### How should this be manually tested?
1. Clone the repo.
2. Follow the instructions on how to set up the repo in the ReadMe.
3. Check to see if everything works after the code refactoring.
